src/commands/new.ts

```diff
@@ -515,17 +509,6 @@ export default {
     stopSpinner(renameSpinnerMsg, "🎨")
     // #endregion
 
-    // #region Install CocoaPods
-    // install pods
-    if (shouldFreshInstallDeps) {
-      startSpinner("Baking CocoaPods")
-      await spawnProgress(`npx pod-install@${deps.podInstall}`, {
```

Comment:
Simply remove `npx pod-install` during new command since it would have just been called on post install from the package manager command